src/components/password/password.test.tsx

```diff
@@ -0,0 +1,204 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import Password from "./password.component";
+import guid from "../../__internal__/utils/helpers/guid";
+
+const mockedGuid = "guid-12345";
+jest.mock("../../__internal__/utils/helpers/guid");
+
+(guid as jest.MockedFunction<typeof guid>).mockImplementation(() => mockedGuid);
+
+test("should render a text input on render with type 'password' on render", () => {
```

Comment:
**nit-pick(non-blocking):** doubled up on the 'on render'
```suggestion
test("should render a text input with type 'password' on render", () => {
```

Reply:
oops, kept them both to just mention one render now 👍